jenkins-scripts/tools/outdated-job-runner.groovy

```diff
@@ -52,181 +53,33 @@ def runJobsInAvailableNodes(LinkedHashMap outdatedJobs) {
-    'ignition_fuel-tools-ci-gz-fuel-tools9-homebrew-amd64',
-    'ignition_fuel-tools-ci-ign-fuel-tools4-homebrew-amd64',
-    'ignition_fuel-tools-ci-ign-fuel-tools7-homebrew-amd64',
-    'ignition_gazebo-ci-gz-sim7-homebrew-amd64',
-    'ignition_gazebo-ci-gz-sim8-homebrew-amd64',
-    'ignition_gazebo-ci-ign-gazebo3-homebrew-amd64',
-    'ignition_gazebo-ci-ign-gazebo6-homebrew-amd64',
-    'ignition_gui-ci-gz-gui7-homebrew-amd64',
-    'ignition_gui-ci-gz-gui8-homebrew-amd64',
-    'ignition_gui-ci-ign-gui3-homebrew-amd64',
-    'ignition_gui-ci-ign-gui6-homebrew-amd64',
-    'ignition_launch-ci-gz-launch6-homebrew-amd64',
-    'ignition_launch-ci-gz-launch7-homebrew-amd64',
-    'ignition_launch-ci-ign-launch2-homebrew-amd64',
-    'ignition_launch-ci-ign-launch5-homebrew-amd64',
-    'ignition_math-ci-gz-math7-homebrew-amd64',
-    'ignition_math-ci-ign-math6-homebrew-amd64',
-    'ignition_msgs-ci-gz-msgs10-homebrew-amd64',
-    'ignition_msgs-ci-gz-msgs9-homebrew-amd64',
-    'ignition_msgs-ci-ign-msgs5-homebrew-amd64',
-    'ignition_msgs-ci-ign-msgs8-homebrew-amd64',
-    'ignition_physics-ci-gz-physics6-homebrew-amd64',
-    'ignition_physics-ci-gz-physics7-homebrew-amd64',
-    'ignition_physics-ci-ign-physics2-homebrew-amd64',
-    'ignition_physics-ci-ign-physics5-homebrew-amd64',
-    'ignition_plugin-ci-gz-plugin2-homebrew-amd64',
-    'ignition_plugin-ci-ign-plugin1-homebrew-amd64',
-    'ignition_rendering-ci-gz-rendering7-homebrew-amd64',
-    'ignition_rendering-ci-gz-rendering8-homebrew-amd64',
-    'ignition_rendering-ci-ign-rendering3-homebrew-amd64',
-    'ignition_rendering-ci-ign-rendering6-homebrew-amd64',
-    'ignition_sensors-ci-gz-sensors7-homebrew-amd64',
-    'ignition_sensors-ci-gz-sensors8-homebrew-amd64',
-    'ignition_sensors-ci-ign-sensors3-homebrew-amd64',
-    'ignition_sensors-ci-ign-sensors6-homebrew-amd64',
-    'ignition_tools-ci-gz-tools2-homebrew-amd64',
-    'ignition_tools-ci-ign-tools1-homebrew-amd64',
-    'ignition_transport-ci-gz-transport12-homebrew-amd64',
-    'ignition_transport-ci-gz-transport13-homebrew-amd64',
-    'ignition_transport-ci-ign-transport11-homebrew-amd64',
-    'ignition_transport-ci-ign-transport8-homebrew-amd64',
-    'ignition_utils-ci-gz-utils2-homebrew-amd64',
-    'ignition_utils-ci-ign-utils1-homebrew-amd64',
-    'sdformat-ci-sdf12-focal-amd64',
-    'sdformat-ci-sdf13-focal-amd64',
-    'sdformat-ci-sdf14-jammy-amd64',
-    'sdformat-ci-sdformat12-homebrew-amd64',
-    'sdformat-ci-sdformat13-homebrew-amd64',
-    'sdformat-ci-sdformat14-homebrew-amd64',
-    'sdformat-ci-sdformat9-bionic-amd64',
-    'sdformat-ci-sdformat9-homebrew-amd64',
-    'sdformat-sdf-12-win',
-    'sdformat-sdf-13-win',
-    'sdformat-sdf-14-win',
-    'sdformat-sdf-9-win',
-];
-def trackedJobs = new LinkedHashSet(trackedJobsList);
+def JOBS_URL = 'https://raw.githubusercontent.com/Crola1702/release-tools/master/jenkins-scripts/dsl/jobs.txt'
```

Comment:
Please check #1089 to see the final path and final name of the log file generated.